addons/html_editor/static/tests/format/bold.test.js

```diff
@@ -42,37 +42,19 @@ test("should make two paragraphs not bold", async () => {
     });
 });
 
-test("should make qweb tag bold", async () => {
+test("should not make qweb tag bold when its contenteditable false element", async () => {
```

Comment:
@kebeclibre @bastien-ipb I fear this would break the feature in the report editor, right? But in the same time, it's weird that we would be able to edit content that is explicitely set to be non-editable...

Reply:
indeed, this would be a problem for us. The content of the p should not be editable but the attributes of the p should be.
The test you modify was right in the first place.

Reply:
Here, we only want to show the toolbar when the element is contentEditable="false" and it's a `[t-field], [t-out], or [t-esc]`, right? Then why are we checking whether `anchorNode` and `focusNode` are editable for all nodes?

I think we should either:
1. Only check if the selected nodes closest is `[t-field], [t-out], or [t-esc]`.
2. Or check whether `anchorNode` and `focusNode` are editable, but only if the selected nodes closest is `[t-field], [t-out], or [t-esc]`.

Reply:
Ok for option 2. We'll adapt it later if we end up finding other cases for which we need it. Thanks!

Reply:
@dmo-odoo I have applied the changes.

Reply:
@dmo-odoo I mean, since anchorNode and focusNode are going to be editable elements due to this [code](https://github.com/odoo/odoo/blob/18.0/addons/html_editor/static/src/utils/selection.js#L88), do we still need to check whether both anchorNode and focusNode are editable in order to show the toolbar?
Or is there a case I might be missing where either anchorNode or focusNode could be non-editable

Reply:
I would say yes because we could have programmatically set the selection there after this function has executed for example, and we don't want the toolbar to show.